cpp/src/slice2cs/CsUtil.cpp

```diff
@@ -928,22 +928,22 @@ Slice::CsGenerator::writeTaggedMarshalCode(Output &out,
     }
     else if(en)
     {
-        out << nl << "if(" << param << " is " << typeToString(en, scope) << ")";
+        out << nl << "if (" << param << " is " << typeToString(en, scope) << ")";
         out << sb;
         out << nl << stream << ".WriteEnum(" << tag << ", (int)" << param << ".Value);";
         out << eb;
     }
     else if(seq)
     {
-        writeTaggedSequenceMarshalUnmarshalCode(out, seq, scope, param, tag, true, stream);
+        writeTaggedSequenceMarshalCode(out, seq, scope, param, tag, stream);
```

Comment:
Since this function is called only once and is not that long, I think it would be more readable to put the code right there and remove the helper function.

Reply:
Fixed